ci/test_python_distributed.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: BSD-3-Clause
+
+set -euo pipefail
+
+source "$(dirname "$0")/test_common.sh"
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key test_python \
```

Comment:
@pentschev The errors might just be an incorrect file key?
```suggestion
  --file_key test_python_distributed \
```